### PR TITLE
Makes the clown play a sad trombone sound on death

### PR DESCRIPTION
**Pull request by unid15**
Port of https://github.com/tgstation/tgstation/pull/18323

WARNING: untested I still have to test this

### CODE REVIEW

**Comment by ComicIronic** [comment]
Seems very exploitable.

**Comment by 9600bauds** [comment]
So now the clown has an unobtainable implant upon spawn?
Could shit in your cornflakes if you try to impersonate the clown I guess, doubly so if it shows up in SecHUDs
At the very least have another spawn in the theater backstage

**Comment by Exxion** [comment]
IT WON'T LET ME ADD ANY REACTIONS

**Comment by NigglyWiggly** [comment]
Couldn't you just make this happen when someone with the job var set to clown dies, instead of making it an implant

**Comment by Exxion** [comment]
No, because that would be even worse than just having an unobtainable implant

**Comment by 9600bauds** [comment]
Tie it to the clown's PDA?

**Comment by AI-Is-Rogue** [comment]
This is a horrible idea, but. What if the trombone noise played on the clown's PDA and any other PDA the clown sent viruses to when the clown dies.

**Comment by 9600bauds** [comment]
That is so fucking terrible

I love it

**Comment by JustSumBody** [comment]
@AI-is-rogue
Like life alert, but for clowns instead of old people.

File: code/game/objects/items/weapons/implants/implantcase.dm [hunk 1]
```diff
@@ -128,3 +128,15 @@
 		src.imp = new /obj/item/weapon/implant/death_alarm( src )
 		..()
 		return
+
+
+/obj/item/weapon/implantcase/trombone
+	name = "Glass Case- 'Sad Trombone'"
```
Comment (PJB3005) [inline code/game/objects/items/weapons/implants/implantcase.dm:134]: Needs capitalization fixed.

File: code/game/objects/items/weapons/implants/implantcase.dm [hunk 2]
```diff
@@ -128,3 +128,15 @@
...
+	desc = "A case containing a sad trombone implant."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "implantcase-b"
+
+/obj/item/weapon/implantcase/trombone/New()
+	src.imp = new /obj/item/weapon/implant/trombone( src )
+	..()
+	return
```
Comment (PJB3005) [inline code/game/objects/items/weapons/implants/implantcase.dm:142]: Wew

**Comment by icantthinkofanameritenow** [comment]
Does the base death alert implant even work?

And why don't we have it in-game?

**Comment by 9600bauds** [comment]
We have something even better, it's called SPS

**Comment by ihadtoregisterforthis** [comment]
What about gun suicide sad song?
Would one get overridden or mixed in an unpleasant noise?

**Comment by ghost** [comment]
The Death Alarm implant doesn't work, or at least is no longer actively used in game

I feel it would be better if the Clown's PDA had an app that made it play a sad trombone tune along with all infected PDAs when the Clown dies

**Comment by ghost** [comment]
I'd prefer it to be tied to the PDA.

**Comment by ScrambledSteak** [comment]
If you infect somebody more than once, it should play the sound effect even louder.

Keep slipping somebody, while sending 50 viruses to captain comdom who wouldn't let you get your spare, then kill yourself.

**Comment by NigglyWiggly** [comment]
At 50 viruses it should make the infected PDA explode while the trombone plays

**Comment by AI-Is-Rogue** [comment]
Make it so every person you slip on the PDA gets you another virus, but you can only get one virus per unique slip, and it doesn't work on monkeys or monkey-men.

**Comment by unid15** [comment]
meme pr goodbye

**Pull request closed without merging**